docker/caffe2/jenkins/common/install_devtoolset.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -ex
+
+[ -n "$CENTOS_VERSION" ]
+
+yum install -y centos-release-scl
+yum install -y devtoolset-$CENTOS_VERSION
```

Comment:
wait I think CENTOS_VERSION and devtoolset version are decoupled.

Reply:
Yes, somewhat but let me do this more proper.